apps/developer-hub/src/app/(homepage)/page.tsx

```diff
@@ -0,0 +1,3 @@
+import { Homepage } from "@/src/components/Pages/Homepage";
```

Comment:
In cases like this I recommend a simple re-export as webpack will optimize away the module:

```ts
export { Homepage as default } from "..."
```